homeassistant/components/devolo_home_control/__init__.py

```diff
@@ -76,6 +80,22 @@ def shutdown(event):
     return True
 
 
+async def async_migrate_entry(hass, entry: ConfigEntry) -> bool:
```

Comment:
It's good if we can avoid migrating the config entry version since it's not possible to roll back after version bump.

Can we identify the old unique_id and update the config entry unique_id in that case?

Reply:
Yes, while the new unique_id would be a uuid, the old one is a 16-digit number. Do you have a pointer to docs how to change the unique_id without migration (or an other integration that I can study)?

Reply:
Like this:
https://github.com/home-assistant/core/blob/69710cb6647b65308697453545ac8e07f60558b5/homeassistant/components/hue/__init__.py#L112-L113

Just adapt the check.

Reply:
Thanks. This makes a lot less code needed.